core/src/main/scala/akka/kafka/scaladsl/Consumer.scala

```diff
@@ -171,5 +172,9 @@ object Consumer {
     }
   }
 
+  def committablePartitionedSource[K, V](settings: ConsumerSettings[K, V]):
+    Source[(TopicPartition, Source[CommittableMessage[K, V], Control]), Control] = {
```

Comment:
@patriknw should we wait for comments from @drewhk or just start implementation?